upsint/cli.py

```diff
@@ -105,7 +106,20 @@ def create_pr(target_remote, target_branch):
 
     git_push()
 
-    title, body = prompt_for_pr_content(get_commit_msgs(base))
+    commit_msgs = get_commit_msgs(base)
+    project_pr_template = None
+    if "github.com" in url:
```

Comment:
I would probably do
```suggestion
    if "github.com" in git_project.get_web_url():
```
, since you can have remote set up via SSH with alias.